server/src/test/resources/permissions.json

```diff
@@ -13,5 +13,10 @@
       "hono-client": [ "READ", "WRITE" ],
       "rest-adapter": [ "READ", "WRITE" ],
       "mqtt-adapter": [ "READ" ]
+  },
+  "credentials/DEFAULT_TENANT": {
+    "hono-client": [ "READ", "WRITE" ],
+    "rest-adapter": [ "READ", "WRITE" ],
```

Comment:
I think the adapters will need both READ and WRITE in order to interact with the credentials service, one link for sending the request (=WRITE) and another one for receiving the response (=READ)